src/languages/scala.js

```diff
@@ -112,6 +112,30 @@ export default function(hljs) {
     contains: [ NAME ]
   };
 
+  const EXTENSION = {
+    begin: [
+      /^\s*/, // Is first token on the line
+      'extension',
+      /\s+(?=[[(])/, // followed by at least one space and `[` or `(`
```

Comment:
Should I use escapes for `(` and `[` as in the other PR?
The rule would be `/\s+(?=\[|\()/`.

Reply:
This is fine, it's a character group.  But we don't generally use a single character group to avoid escapes, hence my flagging the other.